src/Centreon/Infrastructure/PlatformTopology/PlatformTopologyRepositoryRDB.php

```diff
@@ -171,4 +172,26 @@ public function findMonitoringIdFromName(string $serverName, bool $localhost): ?
 
         return $platformTopology;
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function findLocalhostMonitoringName(): ?string
```

Comment:
Do not create a method only to retrieve its name.
Return an entity and then retrieve its name

Reply:
You are out of your business context.
If you need to retrieve a monitoring server entity, you should use the MonitoringServerService
(MonitoringServerService::findLocalServer)